spec/services/cleaners/advocate_claim_cleaner_spec.rb

```diff
@@ -1,6 +1,42 @@
 require 'rails_helper'
 
-describe Cleaners::AdvocateClaimCleaner do
+RSpec.shared_examples 'delete fixed fees' do
+  it { expect { call_cleaner }.to change { claim.fixed_fees.size }.to 0 }
+end
+
+RSpec.shared_examples 'does not delete fixed fees' do
+  it { expect { call_cleaner }.not_to(change { claim.fixed_fees.size }) }
+end
+
+RSpec.shared_examples 'does not delete misc fees' do
+  it { expect { call_cleaner }.not_to(change { claim.misc_fees.size }) }
+end
+
+RSpec.shared_examples 'clear basic fees' do
+  it { expect { call_cleaner }.not_to(change { claim.basic_fees.size }) }
+  it { expect { call_cleaner }.to change { claim.basic_fees.flat_map(&:dates_attended).size }.to 0 }
+end
+
+RSpec.shared_examples 'does not clear basic fees' do
+  it { expect { call_cleaner }.not_to(change { claim.basic_fees.size }) }
+  it { expect { call_cleaner }.not_to(change { claim.basic_fees.flat_map(&:dates_attended).size }) }
+end
+
+RSpec.shared_examples 'clear cracked details' do
+  it { expect { call_cleaner }.to change(claim, :trial_fixed_notice_at).to nil }
+  it { expect { call_cleaner }.to change(claim, :trial_fixed_at).to nil }
+  it { expect { call_cleaner }.to change(claim, :trial_cracked_at).to nil }
+  it { expect { call_cleaner }.to change(claim, :trial_cracked_at_third).to nil }
+end
+
+RSpec.shared_examples 'does not clear cracked details' do
```

Comment:
nit - as with other negative tests I am wary of false positives and like think about if there is positive way this could be achieved

Reply:
I'll have a look at this. Would a `expect { ... }.not_to change(...).from(old_value)` satisfy?